test/test_parse_de.py

```diff
@@ -16,10 +16,12 @@
 import unittest
 from datetime import datetime, time
 
+from lingua_franca import load_language
 from lingua_franca.parse import extract_datetime
 from lingua_franca.parse import extract_number
 from lingua_franca.parse import normalize
 
+load_language('de')
```

Comment:
As I recall, it's that I only *added* setup methods to those languages that were having trouble due to the order in which tests were run, and the need to un/load each language for testing.

I could be misremembering. Either way, this can be standardized, sure thing!